inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/SPARQLQueryStore.java

```diff
@@ -0,0 +1,31 @@
+package de.tudarmstadt.ukp.inception.kb;
+
+public class SPARQLQueryStore
+{
+    
+    public static String aLimit = "1000";
+    public static String PROPERTYLIST_RDF_QUERY = String.join("\n"
+            , "SELECT DISTINCT ?s ?l WHERE {"
+            , "  { ?s ?pTYPE ?oPROPERTY .}"
+            , "  UNION "
+            , "  { ?s a ?prop" 
+            , "    FILTER regex(str(?prop), \"Property$\") }"
```

Comment:
@reckart  Here, I didn't want to add all of the types of rdf:Property such as ObjectProperty, DataTypeProperty, AnnotationProperty, etc.. as they might have increased the time complexity. But still not sure about it, in case there can be better way to avoid this. If you have any inputs, please do comment

Reply:
Well, we certainly should not use a regular expression for matching here. I think a better way to do this and to give the KB an opportunity to optimize the query is using the `VALUES` keyword:

```
VALUES ?labelpredicate {rdfs:label skos:altLabel}
```

Cf. `de.tudarmstadt.ukp.inception.conceptlinking.util.QueryUtil`